loopback4/package.json

```diff
@@ -42,6 +43,7 @@
   ],
   "dependencies": {
     "@contrast/test-bench-utils": "3.15.0",
+    "@contrast/loopback-test-bench": "3.15.0",
```

Comment:
this gets really weird now because @contrast/test-bench-utils isn't really needed in this repo now. It also takes forever to rewrite because it essentially is rewriting the test bench utils twice. once here and once in the loopback 3 app.